### ISSUE TITLE
FlatColor material

### ISSUE THREAD

**Issue by jasperro** [deviantfero/wpgtk-templates#12]
I've made a fork of your FlatColor theme in this repo: [PlatColor](https://github.com/jasperro/PlatColor). I've edited it to have ripple animations, buttons with shadows and tabs with underlines. I don't know if this is the direction you want to take the theme in, but if it is, please tell me. It's still rough around the edges, and not ready to be merged. I've only edited the gtk-3.20 files so far.
![screenshot_2019-10-19-003328](https://user-images.githubusercontent.com/42558625/67132226-4737a500-f208-11e9-8294-29eb3dbad2ff.png)

**Comment by deviantfero** [comment]
we can have both, I just need to support it in the `wpg-install.sh` script, that shouldn't be that hard once we're done, that's the good thing about wpgtk, since they're only templates it really doesn't break anything :+1:, keep up the good work, let me know when it's finished.

**Comment by deviantfero** [comment]
Any updates on this? is it ready for primetime?

**Comment by jasperro** [comment]
Not really, I'm back to using FlatColor. I might look into continuing developing the theme further when I have the time. I guess the theme works, but it still has a lot of improving left to do.

**Comment by niksingh710** [comment]
Any Updates?

**Comment by deviantfero** [comment]
Would love to hear from this as well, I haven't had time to find a new
theme to work from, and I might not be able to keep up with GTK, if anyone
has specific bugs on it, you can find more info on that by using the GTK
inspector.

That would help a lot when reporting bugs, maybe I can add that to the
README.md

**Comment by jasperro** [comment]
I am currently on KDE, not using wpgtk anymore.

I haven't done anything with the theme for a long time, and have archived my repository. The theme never has been more than me just tinkering with the gtk css, and is still in the same unfinished state. Maybe one of the theme forks could be used, like https://github.com/bR3iN/FlatColor-16?

With gtk4 being used in more places, and libadwaita becoming the standard for a lot of gtk apps, I think it is a good idea to do it like https://github.com/GradienceTeam/Gradience to theme gtk4 and gtk3 only by recoloring the default Adwaita theme. Maybe this is worth looking into instead? Should be much easier to maintain.
